maven/fetch.bzl

```diff
@@ -24,8 +24,6 @@ def _fetch_artifact_impl(ctx):
     ]
     for repo, url in ctx.attr.repository_urls.items():
         args.append("--repository=%s|%s" % (repo, url))
-    if verbosity > 1:
```

Comment:
It isn't. It's a second call - that' snot boolean, it's a verbosity level. 0, 1, 2. Basically how many "--versions" do you send to the client, which interprets those each as a count of verbosity.